packages/builder/src/pages/builder/app/[application]/design/[screenId]/[componentId]/_components/Component/ComponentSettingsSection.svelte

```diff
@@ -177,9 +177,10 @@
               defaultValue={setting.defaultValue}
               nested={setting.nested}
               onChange={val => updateSetting(setting, val)}
-              highlighted={$componentStore.highlightedSettingKey ===
-                setting.key}
-              propertyFocus={$componentStore.propertyFocus === setting.key}
+              highlighted={$builderStore.highlightedSetting?.key === setting.key
```

Comment:
Can this not just be 
```js
highlighted={$builderStore.highlightedSetting?.key === setting.key}
```
`highlighted` is a boolean prop, so we just need to check if the keys match.